app/views/layouts/application_mailer.html.erb

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html>
+  <head>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
+    <%= stylesheet_link_tag "unimported/email" %>
+    <%= yield :head %>
+  </head>
+  <body>
+    <%= yield :body %>
```

Comment:
Use a plain `yield` here with no name, then you don't need to put `content_for :body` into every single view, same as with a regular web UI layout.